app/views/votes/_voter.html.erb

```diff
@@ -1,7 +1,7 @@
 <% if current_user %>
 <div class="vote-arrow pull-left"><!-- class: glyphicon ... is from bootstrap.  see: (http://getbootstrap.com/components/) -->
-  <div><%= link_to " ", post_up_vote_path(post), class: 'glyphicon glyphicon-chevron-up', method: :post  %></div>
+  <div><%= link_to " ", post_up_vote_path(post), class: 'glyphicon glyphicon-chevron-up', method: :post, remote: true  %></div>
   <div><strong><%= post.points %></strong></div>
-  <div><%= link_to " ", post_down_vote_path(post), class: 'glyphicon glyphicon-chevron-down', method: :post %></div>
+  <div><%= link_to " ", post_down_vote_path(post), class: 'glyphicon glyphicon-chevron-down', method: :post, remote: true %></div>
 </div>
 <% end %>
```

Comment:
In here, you will want the `<div>` displaying the `post.points` to be updated accordingly to the clicks on either link. In order to be updated, it has to be selectable, so it is a good idea assigning a unique CSS identifier to it—let's use the `post` id attribute:

```diff
-    <div><strong><%= post.points %></strong></div>
+    <div id="post-<%= post.id %>-vote"><strong><%= post.points %></strong></div> 
```

Then, as a side-effect to clicking the up vote or down vote links, in the JavaScript templates which will be sent back to the browser, you will instruct the browser to run JS code to update the points count (see below).